src/services/SendEmailProcessor.ts

```diff
@@ -31,7 +31,7 @@ export class SendEmailProcessor {
 		this.validationHelper = new ValidationHelper();
 		this.metrics = metrics;
 		this.govNotifyService = SendEmailService.getInstance(this.logger, GOVUKNOTIFY_API_KEY);
-		this.environmentVariables = new EnvironmentVariables(logger, ServicesEnum.SEND_EMAIL_PROCESSOR_SERVICE);
```

Comment:
is SEND_EMAIL_PROCESSOR_SERVICE still needed in ServicesEnum?

Reply:
Removed it from the enum. Thanks